backend/api/views.py

```diff
@@ -1,8 +1,77 @@
-from django.shortcuts import render
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from .serializers import FiboUserSerializer
+from .models import FiboUser
+from django.contrib.auth import authenticate
 
-# Create your views here.
-from django.http import HttpResponse
 
+@api_view(['GET'])
+def getUser(request, pk):
+    user = FiboUser.objects.get(id=pk)
+    serializer = FiboUserSerializer(user, many=False)
+    return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
I've created a [backlog entry](https://dhbw-karlsruhe.myjetbrains.com/youtrack/issue/FiBo-161/Put-users-id-behind-authentication-authorization) to make sure we only allow to return [PII] when authenticated + authorized.

[PII]: https://en.wikipedia.org/wiki/Personal_data